k8s/migratev2/templates/staging-claim.yaml

```diff
@@ -4,8 +4,6 @@ metadata:
   name: "{{.Release.Name}}-staging-claim"
 spec:
   storageClassName: {{ .Values.storageClassName }}
-  persistentVolumeReclaimPolicy: Delete
```

Comment:
this one should not be deprecated. Why do you want to remove this?

Reply:
reverted